src/components/Reports/SubscriberHistory/SubscriberHistoryCurrentSearch/SubscriberHistoryCurrentSearch.js

```diff
@@ -30,26 +34,33 @@ const SubscriberHistoryCurrentSearch = ({ email, dependencies: { dopplerApiClien
       {state.loading ? (
         <Loading />
       ) : state.subscriber ? (
-        <table className="dp-c-table">
-          <thead>
-            <tr>
-              <th>email</th>
-              <th>nombre</th>
-              <th>apellido</th>
-              <th>ranking</th>
-              <th>status</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr>
-              <td>{state.subscriber.email}</td>
-              <td>{state.subscriber.firstName ? state.subscriber.firstName.value : ''}</td>
-              <td>{state.subscriber.lastName ? state.subscriber.lastName.value : ''}</td>
-              <td>{state.subscriber.score}</td>
-              <td>{state.subscriber.status}</td>
-            </tr>
-          </tbody>
-        </table>
+        <>
+          <table className="dp-c-table">
+            <thead>
+              <tr>
+                <th>email</th>
+                <th>nombre</th>
+                <th>apellido</th>
+                <th>ranking</th>
+                <th>status</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <span onClick={showSubscriberCampaigns}>{state.subscriber.email}</span>
```

Comment:
Herne perhaps you could add underline to style or green color to identify as link. What do you think?

Reply:
yes i know! it's only for working. When apply the styles i add this. Thanks!

Reply:
Maybe, it could be a real link. I mean, it would be great if we could have permalink for the different pages.

Reply:
Creo que este tema es importante, por favor si no se entiende lo quepropongo hablemosló.

Es el tipo de cosas que si no hacemos bien desde el comienzo, terminan sin arreglarse nunca y embrollandose.

Reply:
si lo entiendo, pero si es un link real deberia tener una url tambien, decis la misma url parametrizada?

Reply:
claro, pienso que tendríamos que pensar en tener todas las URLs que podamos.